Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/PersonStats.hs

```diff
@@ -0,0 +1,160 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Domain.Action.UI.PersonStats where
+
+import Data.Time hiding (getCurrentTime)
+import qualified Domain.Types.Merchant as Merchant
+import qualified Domain.Types.Person as DP
+-- import Kernel.Storage.Esqueleto (runInReplica)
+import Kernel.Beam.Functions
+import Kernel.External.Encryption
+import Kernel.Prelude
+-- import qualified Kernel.Storage.Esqueleto as Esq
```

Comment:
Can you remove these comments(are present at other places too)

Reply:
I haven't removed these comments  because KV. branch also have comments of Esq and they want them for reference, I've  discussed with them before leaving the comments

Reply:
Sure, you can have it in the queries for reference but you can remove from other places and also these imports.